crates/dprint/src/utils/glob_utils.rs

```diff
@@ -81,8 +90,12 @@ pub fn to_absolute_glob(pattern: &str, dir: &str) -> String {
   }
 
   // make glob absolute
-  if !is_absolute_path(&pattern) {
-    pattern = glob_join(dir, pattern);
+  if !is_absolute_pattern(&pattern) {
+    if is_relative || pattern.starts_with("**/") || pattern.trim().is_empty() {
+      pattern = glob_join(dir, pattern);
+    } else {
+      pattern = glob_join(dir, format!("**/{}", pattern));
```

Comment:
this will break with negated globs

Reply:
Thanks! I didn't think about negated globs at all.